content/api/cypress-api/config.md

```diff
@@ -140,14 +140,14 @@ Cypress.config() // => {defaultCommandTimeout: 10000, viewportHeight: 900, ...}
 
 ### Not all config values can be changed at all times
 
-Some configuration values cannot be changed while running a test. Anything
+Some configuration values are readonly and cannot be changed while running a test. Anything
 that's not directly under Cypress's control - like timeouts, `userAgent`, or
-environment variables - will be ignored at run-time.
+environment variables - will be ignored at run-time. Be sure to review the list of
+[test-time configuration options](/guides/references/configuration##Test-time-overrides)
 
-### Test Configuration
+## Test Configuration vs `Cypress.config()`
```

Comment:
Changing this heading may have broken other links in the docs - need to look for anywhere that might be references `content/api/cypress-api/config#Test-Configuration` and fix all of those links.

Reply:
There are no links to `content/api/cypress-api/config#Test-Configuration`